crates/abq_cli/tests/cli.rs

```diff
@@ -5171,3 +5574,446 @@ fn retry_continued_manifest_read_on_worker_death() {
         );
     }
 }
+
+#[test]
+#[with_protocol_version]
+#[serial]
+fn test_with_org_access_token_and_implicit_inferred_run_id() {
```

Comment:
This exercises ORG + run ID from env var => always use result of remote configuration